gearcmd/worker.go

```diff
@@ -40,11 +40,17 @@ func (conf TaskConfig) Process(job baseworker.Job) ([]byte, error) {
 	}
 	if err != nil {
 		data["error_message"] = err.Error()
-		log.Printf(kayvee.FormatLog("gearcmd", "error", "END_WITH_ERROR", data))
+		data["type"] = "gauge"
+		data["value"] = 0
+		data["success"] = false
```

Comment:
Cool, sounds good. Would it be easier to add a new log line instead of modifying an existing log line + a toml file? I don't think I have a preference.